modules/userId.js

```diff
@@ -298,6 +300,13 @@ function initSubmodules(submodules, consentData) {
     } else if (submodule.config.value) {
       // cache decoded value (this is copied to every adUnit bid)
       submodule.idObj = submodule.config.value;
+    } else {
+      let result = submodule.submodule.getId(submodule.config.params, consentData);
```

Comment:
If `result` is not reassigned, should be a `const`

Reply:
Done